optimum/modeling_base.py

```diff
@@ -380,27 +380,35 @@ def from_pretrained(
                 )
             model_id, revision = model_id.split("@")
 
+        all_files, _ = TasksManager.get_model_files(
+            model_id,
+            subfolder=subfolder,
+            cache_dir=cache_dir,
+            revision=revision,
+            token=token,
+        )
+
+        config_folder = subfolder
+        if cls.config_name not in all_files:
+            logger.info(
+                f"{cls.config_name} not found in the specified subfolder {subfolder}. Using the top level {cls.config_name}."
+            )
```

Comment:
```suggestion
            logger.debug(
                f"{cls.config_name} not found in the specified subfolder {subfolder}. Using the top level {cls.config_name}."
            )
```
Any chance I can convince you to reduce this to `debug`?

I recognize that it's taken from https://github.com/huggingface/optimum/blob/2c0476eda1398b9a81cb966c817a460ed6e53413/optimum/modeling_base.py#L267-L269 

where `info` is used, but I feel like the setting is a bit different. This new `info` will be triggered every time someone loads an ONNX or OpenVINO model with Sentence Transformers (as those automatically save their modeling files in subfolders away from the configuration). 

Another reason that I'd like it to be reduced to `debug` is because (for some reason?) the `info`-level logs are shown by default in Optimum. Or do I just have a weird/non-normal setup? I don't think I'm setting `set_verbosity_info()` or `TRANSFORMERS_VERBOSITY`.